src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -99,6 +100,7 @@
     <!-- TRANSITIONS -->
     <exit-transition transition_id="to_be_sampled"/>
     <exit-transition transition_id="no_sampling_workflow"/>
+    <exit-transition transition_id="listing_multi_results"/>
```

Comment:
This `listing_multi_results` transition should not be possible in `sample_registered` status. Even if analyses are meant to be captured at point of collection (field) and the sampling workflow is enabled, the sample still needs to be collected first (transition `to_be_sampled`).